### DISCUSSION TITLE
Start up problems

### DISCUSSION THREAD

**Discussion by eIronmoon** [mcguirepr89/BirdNET-Pi#1027] [General]
Hi all--sorry, first time trying a BirdNet install and having some issues.  I'm flashing the 64 lite OS, and able to successfully pull the installation as outlined in the installation guide.  However when I try to remote in via either the IP or the host name, I get the following landing page from Caddy, where I believed it would boot directly into the BirdNet GUI.   Any help would be appreciated as I wasn't finding this same discussion.  Thanks!

> Your web server is working. Now make it work for you. 💪
> 
> Caddy is ready to serve your site over HTTPS:
> 
>     Point your domain's A/AAAA DNS records at this machine.
>     Upload your site's files to /var/www/html.
>     Edit your Caddyfile at /etc/caddy/Caddyfile:
>         Replace :80 with your domain name
>         Change the site root to /var/www/html
>     Reload the configuration: systemctl reload caddy
>     Visit your site!
> 
> If that worked 🥳
> 
> Awesome! You won't have to look at this slanted page anymore.
> 
> Remember, Caddy can do a lot more than serve static files. It's also a powerful reverse proxy and application platform. You can use the Caddyfile to enable any other features you need. Or you could use Caddy's API to configure it programmatically.
> 
> Everything you need to know is either in the [📖 Caddy documentation](https://caddyserver.com/docs/) or the manual for your OS/platform. Have fun!
> If that didn't work 😶
> 
> It's okay, you can fix it! First check the following things:
> 
>     Service status: systemctl status caddy
>     Logs: journalctl --no-pager -u caddy
>     Are your site's files readable by the caddy user and group? ls -la /var/www/html
>     Is the caddy home directory writeable? ls -la /var/lib/caddy
>     Ensure your domain's A and/or AAAA records point to your machine's public IP address: dig example.com
>     Are your ports 80 and 443 externally reachable, and is Caddy able to bind to them? Check your firewalls, port forwarding, and other network configuration.
> 
> WAIT! Before trying again, switch to [Let's Encrypt's staging environment](https://letsencrypt.org/docs/staging-environment/) to avoid being accidentally [rate limited](https://letsencrypt.org/docs/rate-limits/). Once you get everything else working, it's safe to switch back.
> 
> Depending on your DNS provider, it may take a while for the DNS records to propagate. Even when everything is configured properly, automated challenges to obtain TLS certificates usually take several seconds, but may take up to several minutes or hours.
> 
> If you still need help, we have a [great community](https://caddy.community/)! First [try a search](https://caddy.community/search), and if your question is original, go ahead and ask it! Remember to pay it forward and help others too. 😁
> 
> Visit Caddy on: [GitHub](https://github.com/caddyserver/caddy) or [Twitter](https://twitter.com/caddyserver) or [Our Forum](https://caddy.community/)

**Comment by dzaveta** [comment]
Hi. Same issue. Any pointers on how to resolve this?

**Comment by Hypseus** [comment]
I had the same problem. My mistake was that I had installed the "**bookwoorm**" version of Raspberry OS.

1. Check the installation log (`~/installation-<timestamp>.txt`) to see if there is no error noted at the end of the file.
2. Check your Raspberry OS version:
```
birdnetpi:~$ grep VERSION /etc/os-release 
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
```
You must use the version "bullseye".

**Comment by cartelcar** [comment]
Hi, same issue here, however I confirmed that bullseye is installed. Any other hints how I could fix it?

**Comment by dzaveta** [comment]
I ended up using the version of bullseye that was used in this video (4-4-2022) https://youtu.be/IM-F4sJ-5rc?si=JI-6CK3J-EvTep-B and that cleared up all my issues.

**Comment by Svardsten53** [comment]
This Bullseye file works and have the last updates of the OS.

`2023-05-03-raspios-bullseye-arm64-lite.img.xz`
